lexicons/com/atproto/sync/subscribeRepos.json

```diff
@@ -19,6 +19,7 @@
           "type": "union",
           "refs": [
             "#commit",
+            "#resync",
```

Comment:
As I recall we prospectively landed on `#sync` here.

Reply:
yep! updated